generator/Makefile

```diff
@@ -129,6 +129,7 @@ mibs: \
 $(MIBDIR)/apc-powernet-mib:
 	@echo ">> Downloading apc-powernet-mib"
 	@curl $(CURL_OPTS) -o $(MIBDIR)/apc-powernet-mib "$(APC_URL)"
+	@sed -i.bak -E 's/(DisplayString[[:space:]]*FROM )RFC1213-MIB/\1SNMPv2-TC/' $(MIBDIR)/apc-powernet-mib
```

Comment:
We don't want to make a backup file, as net-snmp reads all files regardless of filename. This will likely still cause generating errors. It's fine to apply this with no backup.

```suggestion
	@sed -i -E 's/(DisplayString[[:space:]]*FROM )RFC1213-MIB/\1SNMPv2-TC/' $(MIBDIR)/apc-powernet-mib
```

Reply:
Since this is a bit of a hack, can you add a comment about the workaround?

Reply:
I included `.bak` for two reasons: 

1. to be consistent with where it's used elsewhere: https://github.com/prometheus/snmp_exporter/blob/v0.21.0/generator/Makefile#L208
2. because some version of sed behave strangely without it. On a Mac it will create "filename-e"

```
MacBook-Pro-6:~ $ mkdir tmp
MacBook-Pro-6:~ $ cd tmp
MacBook-Pro-6:tmp $ echo hello >foo
MacBook-Pro-6:tmp $ sed -i -e 's/hello/goodbye/' foo
MacBook-Pro-6:tmp $ ls -l
total 16
-rw-r--r--  1 brian  staff  8  1 May 16:58 foo
-rw-r--r--  1 brian  staff  6  1 May 16:58 foo-e
MacBook-Pro-6:tmp $
```

And you can't provide an empty extension:

```
MacBook-Pro-6:tmp $ sed -i'' -e 's/hello/goodbye/'
sed: -I or -i may not be used with stdin
MacBook-Pro-6:tmp $ sed -i '' -e 's/hello/goodbye/'
sed: -I or -i may not be used with stdin
```

(maybe FreeBSD would be the same, I've not tested)

But if you prefer, I could change both instances to `@sed ... >file.new && mv file.new file` or similar?

Reply:
Ugh, MacOS. What about sed then rm?

Reply:
Could also change the command to `curl | sed`

Reply:
> Could also change the command to curl | sed

That would work for apc-powernet-mib, but not for kemp and cyberpower since those are unpacking zip files.